repose-aggregator/components/filters/openstack-identity-v3/src/test/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3APITest.scala

```diff
@@ -272,6 +323,53 @@ class OpenStackIdentityV3APITest extends FunSpec with BeforeAndAfter with Matche
 
       verify(mockDatastore).put(argThat(equalTo("IDENTITY:V3:TOKEN:test-subject-token")), any[Serializable], intThat(lessThanOrEqualTo((expirationTime.getMillis - currentTime.getMillis).toInt)), any[TimeUnit])
     }
+
+    val statusCodes = List(HttpServletResponse.SC_REQUEST_ENTITY_TOO_LARGE, FilterDirector.SC_TOO_MANY_REQUESTS)
+    statusCodes.foreach { statusCode =>
+      describe(s"should return an Exception when receiving $statusCode and") {
+        it(s"not having headers while retrieving admin token") {
```

Comment:
I removed the variable from the string when I had to split this into separate tests.
Fixed.